lib/go/test/market_test.go

```diff
@@ -47,7 +47,7 @@ func TestMarketDeployment(t *testing.T) {
 	// Should be able to deploy the MetadataViews contract
 	// as a new account with no keys.
 	metadataViewsCode, _ := DownloadFile(MetadataViewsContractsBaseURL + MetadataViewsInterfaceFile)
-	parsedMetadataContract := strings.Replace(string(metadataViewsCode), MetadataReplaceAddress, "0x"+emulatorFTAddress, 1)
+	parsedMetadataContract := strings.Replace(strings.Replace(string(metadataViewsCode), MetadataReplaceAddress, "0x"+emulatorFTAddress, 1), NFTReplaceAddress, "0x"+nftAddr.Hex(), 1)
```

Comment:
NFT reference was added to the metadataviews contract which was causing tests to fail. This fixes it

Reply:
just realizing this was already fixed in #146